server/src/auth/auth.controller.ts

```diff
@@ -45,6 +46,8 @@ export class AuthController {
   ) {}
 
   @SkipIPCheck()
+  // Only allow 3 login attempts every 5 minutes
+  @Throttle(3, 300)
```

Comment:
would 3 be to low? ie: easy to mistype especially since our demographic is older

Reply:
I think 5 would be a safer number, but 3 is also good as we can easily adjust this if they ask